### PR TITLE
Migrate universal sentence encoder

### PR DESCRIPTION
**Pull request by YueLiu-jina**

### CODE REVIEW

File: encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        print('i am a mocker function')
+        assert len(data.shape) == 1
+        return np.stack([[i for i in range(target_output_dim)]] * data.shape[0])
+
+
+target_output_dim = 512
+test_data = np.array(['it is a good day!', 'the dog sits on the floor.'])
+
+
+@mock.patch('tensorflow_hub.load', return_value=MockModule())
```
Comment (JoanFM) [inline encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py:37]: we can also add real case tests now

**Review by hanxiao — CHANGES_REQUESTED** [review]
need to add at least one non-mocked unit test for real-model test, see our discussion on the Slack

File: encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py (hunk 1 above)
Comment (YueLiu-jina) [reply encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py]: solved

**Comment by YueLiu-jina** [comment]
added non-local test for test_encoding_result, all test pass

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**